setup.py

```diff
@@ -30,7 +30,7 @@
         'tabulate>=0.8,<1.0',
         'Jinja2>=2.11,<4.0',
         'PyYAML>=5.3.0,<7.0',
-        'pydantic>=1.8,<1.10',
+        'pydantic>=2.8.0',
```

Comment:
Should set a maximum of `<3.0` here so we don't accidentally upgrade to a new major version if/when it ever comes out in future.

Reply:
Done